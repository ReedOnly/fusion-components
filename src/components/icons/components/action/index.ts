export { default as SearchIcon } from './SearchIcon';
export { default as DoneIcon } from './DoneIcon';
export { default as MinimizeIcon } from './MinimizeIcon';
