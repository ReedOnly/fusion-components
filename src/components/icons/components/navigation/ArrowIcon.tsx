import * as React from 'react';

import useIcon, { IconProps } from '../../../../hooks/useIcon';

type ArrowIconProps = {
    direction: 'up' | 'down' | 'back' | 'forward' | 'ios-back' | 'ios-forward';
} & IconProps;

const ArrowIcon = ({ direction, ...rest }: ArrowIconProps) => {
    let d: string;
    switch (direction) {
        case 'up':
            d =
                'M13 18.7912V7.62123L17.88 12.5012C18.27 12.8912 18.91 12.8912 19.3 12.5012C19.69 12.1112 19.69 11.4812 19.3 11.0912L12.71 4.50123C12.5231 4.31397 12.2695 4.20874 12.005 4.20874C11.7405 4.20874 11.4868 4.31397 11.3 4.50123L4.69997 11.0812C4.30997 11.4712 4.30997 12.1012 4.69997 12.4912C5.08997 12.8812 5.71997 12.8812 6.10997 12.4912L11 7.62123V18.7912C11 19.3412 11.45 19.7912 12 19.7912C12.55 19.7912 13 19.3412 13 18.7912Z';
            break;
        case 'down':
            d =
                'M11.005 5.20874V16.3787L6.12498 11.4987C5.73498 11.1087 5.09498 11.1087 4.70498 11.4987C4.31498 11.8887 4.31498 12.5187 4.70498 12.9087L11.295 19.4987C11.685 19.8887 12.315 19.8887 12.705 19.4987L19.295 12.9087C19.685 12.5187 19.685 11.8887 19.295 11.4987C19.1081 11.3115 18.8545 11.2063 18.59 11.2063C18.3255 11.2063 18.0718 11.3115 17.885 11.4987L13.005 16.3787V5.20874C13.005 4.65874 12.555 4.20874 12.005 4.20874C11.455 4.20874 11.005 4.65874 11.005 5.20874Z';
            break;
        case 'back':
            d =
                'M18.7912 11.005H7.62124L12.5012 6.12502C12.8912 5.73502 12.8912 5.09502 12.5012 4.70502C12.3144 4.51777 12.0608 4.41254 11.7962 4.41254C11.5317 4.41254 11.2781 4.51777 11.0912 4.70502L4.50124 11.295C4.11124 11.685 4.11124 12.315 4.50124 12.705L11.0912 19.295C11.4812 19.685 12.1112 19.685 12.5012 19.295C12.8912 18.905 12.8912 18.275 12.5012 17.885L7.62124 13.005H18.7912C19.3412 13.005 19.7912 12.555 19.7912 12.005C19.7912 11.455 19.3412 11.005 18.7912 11.005Z';
            break;
        case 'forward':
            d =
                'M5.20874 13H16.3787L11.4987 17.88C11.1087 18.27 11.1087 18.91 11.4987 19.3C11.8887 19.69 12.5187 19.69 12.9087 19.3L19.4987 12.71C19.8887 12.32 19.8887 11.69 19.4987 11.3L12.9187 4.70002C12.7319 4.51276 12.4783 4.40753 12.2137 4.40753C11.9492 4.40753 11.6956 4.51276 11.5087 4.70002C11.1187 5.09002 11.1187 5.72002 11.5087 6.11002L16.3787 11H5.20874C4.65874 11 4.20874 11.45 4.20874 12C4.20874 12.55 4.65874 13 5.20874 13Z';
            break;
        case 'ios-back':
            d =
                'M17.0019 2.98499C16.5119 2.49499 15.7219 2.49499 15.2319 2.98499L6.92189 11.295C6.53189 11.685 6.53189 12.315 6.92189 12.705L15.2319 21.015C15.7219 21.505 16.5119 21.505 17.0019 21.015C17.4919 20.525 17.4919 19.735 17.0019 19.245L9.76189 11.995L17.0119 4.74499C17.4919 4.26499 17.4919 3.46499 17.0019 2.98499Z';
            break;
        case 'ios-forward':
            d =
                'M6.99815 21.015C7.48815 21.505 8.27815 21.505 8.76815 21.015L17.0782 12.705C17.4682 12.315 17.4682 11.685 17.0782 11.295L8.76815 2.98499C8.27815 2.49499 7.48815 2.49499 6.99815 2.98499C6.50815 3.47499 6.50815 4.26499 6.99815 4.75499L14.2382 12.005L6.98815 19.255C6.50815 19.735 6.50815 20.535 6.99815 21.015Z';
            break;
        default:
            throw new Error(`Unknown direction ${direction} set on ArrowIcon`);
    }

    const iconFactory = useIcon(<path d={d} fill="currentColor" />);

    return iconFactory(rest);
};

export default ArrowIcon;
