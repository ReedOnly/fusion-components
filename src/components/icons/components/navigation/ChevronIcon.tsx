import * as React from 'react';

import useIcon, { IconProps } from '../../../../hooks/useIcon';

type ChevronIconProps = {
    left: boolean;
} & IconProps;

const ChevronIcon = ({ left, ...rest }: ChevronIconProps) => {
    const d = left
        ? 'M15 6.70502C14.8132 6.51777 14.5595 6.41254 14.295 6.41254C14.0305 6.41254 13.7769 6.51777 13.59 6.70502L9.00002 11.295C8.61002 11.685 8.61002 12.315 9.00002 12.705L13.59 17.295C13.98 17.685 14.61 17.685 15 17.295C15.39 16.905 15.39 16.275 15 15.885L11.12 11.995L15 8.11502C15.39 7.72502 15.38 7.08502 15 6.70502Z'
        : 'M9.00002 6.71063C8.61002 7.10063 8.61002 7.73063 9.00002 8.12063L12.88 12.0006L9.00002 15.8806C8.61002 16.2706 8.61002 16.9006 9.00002 17.2906C9.39002 17.6806 10.02 17.6806 10.41 17.2906L15 12.7006C15.39 12.3106 15.39 11.6806 15 11.2906L10.41 6.70063C10.03 6.32063 9.39002 6.32063 9.00002 6.71063Z';
    const iconFactory = useIcon(<path d={d} fill="currentColor" />);

    return iconFactory(rest);
};

export default ChevronIcon;
