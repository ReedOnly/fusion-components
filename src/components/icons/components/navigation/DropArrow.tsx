import React from 'react';
import { IconProps, useIcon } from '@equinor/fusion-components';

type DropArrowIconProps = {
    direction?: 'up' | 'down' | 'left' | 'right';
} & IconProps;

const DropArrow = ({ direction, ...props }: DropArrowIconProps) => {
    const getIconD = () => {
        switch (direction) {
            case 'up':
                return "M8.70627 12.5862L11.2963 9.99624C11.6863 9.60624 12.3163 9.60624 12.7063 9.99624L15.2963 12.5862C15.9263 13.2162 15.4763 14.2962 14.5863 14.2962H9.40627C8.51627 14.2962 8.07627 13.2162 8.70627 12.5862Z";
            case 'down':
                return "M8.70627 11.4137L11.2963 14.0037C11.6863 14.3937 12.3163 14.3937 12.7063 14.0037L15.2963 11.4137C15.9263 10.7837 15.4763 9.70374 14.5863 9.70374H9.40627C8.51627 9.70374 8.07627 10.7837 8.70627 11.4137Z";
            case 'left':
                return "M12.5862 8.70621L9.99624 11.2962C9.60624 11.6862 9.60624 12.3162 9.99624 12.7062L12.5862 15.2962C13.2162 15.9262 14.2962 15.4762 14.2962 14.5862V9.40621C14.2962 8.51621 13.2162 8.07621 12.5862 8.70621Z";
            case 'right':
                return "M11.4137 15.2917L14.0037 12.7017C14.3937 12.3117 14.3937 11.6817 14.0037 11.2917L11.4137 8.70174C10.7837 8.08174 9.70374 8.52174 9.70374 9.41174V14.5817C9.70374 15.4817 10.7837 15.9217 11.4137 15.2917Z";
            default:
                return "M11.4137 15.2917L14.0037 12.7017C14.3937 12.3117 14.3937 11.6817 14.0037 11.2917L11.4137 8.70174C10.7837 8.08174 9.70374 8.52174 9.70374 9.41174V14.5817C9.70374 15.4817 10.7837 15.9217 11.4137 15.2917Z";
        }
    };
    const iconFactory = useIcon(<path d={getIconD()} fill="currentColor" />);

    return iconFactory(props);
};

export default DropArrow;
