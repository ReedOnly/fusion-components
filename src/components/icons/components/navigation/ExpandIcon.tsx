import * as React from 'react';

import useIcon, { IconProps } from '../../../../hooks/useIcon';

type ExpandIconProps = {
    less: boolean;
} & IconProps;

const ExpandIcon = ({ less, ...rest }: ExpandIconProps) => {
    const d = less
        ? 'M11.295 9.00002L6.70498 13.59C6.31498 13.98 6.31498 14.61 6.70498 15C7.09498 15.39 7.72498 15.39 8.11498 15L12.005 11.12L15.885 15C16.275 15.39 16.905 15.39 17.295 15C17.685 14.61 17.685 13.98 17.295 13.59L12.705 9.00002C12.325 8.61002 11.685 8.61002 11.295 9.00002Z'
        : 'M15.875 9.00001L11.995 12.88L8.11498 9.00001C7.92814 8.81275 7.67449 8.70752 7.40998 8.70752C7.14546 8.70752 6.89181 8.81275 6.70498 9.00001C6.31498 9.39001 6.31498 10.02 6.70498 10.41L11.295 15C11.685 15.39 12.315 15.39 12.705 15L17.295 10.41C17.685 10.02 17.685 9.39001 17.295 9.00001C16.905 8.62001 16.265 8.61001 15.875 9.00001Z';
    const iconFactory = useIcon(<path d={d} fill="currentColor" />);

    return iconFactory(rest);
};

export default ExpandIcon;
