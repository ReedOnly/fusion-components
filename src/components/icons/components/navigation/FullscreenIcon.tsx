import * as React from 'react';

import useIcon, { IconProps } from '../../../../hooks/useIcon';

type FullscreenIconProps = {
    exit: boolean;
} & IconProps;

const FullscreenIcon = ({ exit, ...rest }: FullscreenIconProps) => {
    const d = exit
        ? 'M6 8H8V6C8 5.45 8.45 5 9 5C9.55 5 10 5.45 10 6V9C10 9.55 9.55 10 9 10H6C5.45 10 5 9.55 5 9C5 8.45 5.45 8 6 8ZM8 16H6C5.45 16 5 15.55 5 15C5 14.45 5.45 14 6 14H9C9.55 14 10 14.45 10 15V18C10 18.55 9.55 19 9 19C8.45 19 8 18.55 8 18V16ZM15 19C15.55 19 16 18.55 16 18V16H18C18.55 16 19 15.55 19 15C19 14.45 18.55 14 18 14H15C14.45 14 14 14.45 14 15V18C14 18.55 14.45 19 15 19ZM16 6V8H18C18.55 8 19 8.45 19 9C19 9.55 18.55 10 18 10H15C14.45 10 14 9.55 14 9V6C14 5.45 14.45 5 15 5C15.55 5 16 5.45 16 6Z'
        : 'M7 9C7 9.55 6.55 10 6 10C5.45 10 5 9.55 5 9V6C5 5.45 5.45 5 6 5H9C9.55 5 10 5.45 10 6C10 6.55 9.55 7 9 7H7V9ZM5 15C5 14.45 5.45 14 6 14C6.55 14 7 14.45 7 15V17H9C9.55 17 10 17.45 10 18C10 18.55 9.55 19 9 19H6C5.45 19 5 18.55 5 18V15ZM17 17H15C14.45 17 14 17.45 14 18C14 18.55 14.45 19 15 19H18C18.55 19 19 18.55 19 18V15C19 14.45 18.55 14 18 14C17.45 14 17 14.45 17 15V17ZM15 7C14.45 7 14 6.55 14 6C14 5.45 14.45 5 15 5H18C18.55 5 19 5.45 19 6V9C19 9.55 18.55 10 18 10C17.45 10 17 9.55 17 9V7H15Z';
    const iconFactory = useIcon(
        <path fillRule="evenodd" clipRule="evenodd" d={d} fill="currentColor" />
    );

    return iconFactory(rest);
};

export default FullscreenIcon;
