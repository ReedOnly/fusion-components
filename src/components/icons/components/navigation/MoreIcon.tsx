import * as React from 'react';

import useIcon, { IconProps } from '../../../../hooks/useIcon';

type MoreIconProps = {
    vertical: boolean;
} & IconProps;

const MoreIcon = ({ vertical, ...rest }: MoreIconProps) => {
    const d = vertical
        ? 'M12 8C13.1 8 14 7.1 14 6C14 4.9 13.1 4 12 4C10.9 4 10 4.9 10 6C10 7.1 10.9 8 12 8ZM12 10C10.9 10 10 10.9 10 12C10 13.1 10.9 14 12 14C13.1 14 14 13.1 14 12C14 10.9 13.1 10 12 10ZM10 18C10 16.9 10.9 16 12 16C13.1 16 14 16.9 14 18C14 19.1 13.1 20 12 20C10.9 20 10 19.1 10 18Z'
        : 'M6 10C4.9 10 4 10.9 4 12C4 13.1 4.9 14 6 14C7.1 14 8 13.1 8 12C8 10.9 7.1 10 6 10ZM18 10C16.9 10 16 10.9 16 12C16 13.1 16.9 14 18 14C19.1 14 20 13.1 20 12C20 10.9 19.1 10 18 10ZM10 12C10 10.9 10.9 10 12 10C13.1 10 14 10.9 14 12C14 13.1 13.1 14 12 14C10.9 14 10 13.1 10 12Z';
    const iconFactory = useIcon(
        <path fillRule="evenodd" clipRule="evenodd" d={d} fill="currentColor" />
    );

    return iconFactory(rest);
};

export default MoreIcon;
