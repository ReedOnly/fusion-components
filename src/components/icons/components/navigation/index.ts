export { default as PaginationArrow } from "./PaginationArrow";
export { default as CloseIcon } from "./CloseIcon";