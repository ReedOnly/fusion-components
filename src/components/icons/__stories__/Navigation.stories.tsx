import React from 'react';
import { storiesOf } from '@storybook/react';
import withFusionStory from '../../../../.storybook/withFusionStory';

import { PaginationArrow, DropArrow } from '../components/navigation';

const stories = storiesOf('Icons|Navigation', module);
stories.addDecorator(withFusionStory('Navigation'));

stories.add('Pagination arrows', () => (
    <React.Fragment>
        <PaginationArrow prev />
        <PaginationArrow next />
    </React.Fragment>
));

stories.add('Drop arrows', () => (
    <React.Fragment>
        <DropArrow direction="left" />
        <DropArrow direction="right" />
        <DropArrow direction="up" />
        <DropArrow direction="down" />
    </React.Fragment>
));
