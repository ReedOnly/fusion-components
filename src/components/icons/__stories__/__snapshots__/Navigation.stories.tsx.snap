// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icons|Navigation Arrow Back 1`] = `
<div
  className="container comfortable"
>
  <div
    className="container comfortable"
  >
    <svg
      height={24}
      style={
        Object {
          "color": "",
        }
      }
      viewBox="0 0 24 24"
      width={24}
    >
      <path
        clipRule="evenodd"
        d="M18.7912 11.005H7.62124L12.5012 6.12502C12.8912 5.73502 12.8912 5.09502 12.5012 4.70502C12.3144 4.51777 12.0608 4.41254 11.7962 4.41254C11.5317 4.41254 11.2781 4.51777 11.0912 4.70502L4.50124 11.295C4.11124 11.685 4.11124 12.315 4.50124 12.705L11.0912 19.295C11.4812 19.685 12.1112 19.685 12.5012 19.295C12.8912 18.905 12.8912 18.275 12.5012 17.885L7.62124 13.005H18.7912C19.3412 13.005 19.7912 12.555 19.7912 12.005C19.7912 11.455 19.3412 11.005 18.7912 11.005Z"
        fill="currentColor"
        fillRule="evenodd"
      />
    </svg>
  </div>
  <div
    className="overlay"
  />
  <div
    className="snacks"
  />
  <div
    className="dialog"
  />
</div>
`;

exports[`Icons|Navigation Close 1`] = `
<div
  className="container comfortable"
>
  <div
    className="container comfortable"
  >
    <svg
      height={24}
      style={
        Object {
          "color": "",
        }
      }
      viewBox="0 0 24 24"
      width={24}
    >
      <path
        clipRule="evenodd"
        d="M18.3 5.71002C18.1131 5.52276 17.8595 5.41753 17.595 5.41753C17.3305 5.41753 17.0768 5.52276 16.89 5.71002L12 10.59L7.10997 5.70002C6.92314 5.51276 6.66949 5.40753 6.40497 5.40753C6.14045 5.40753 5.8868 5.51276 5.69997 5.70002C5.30997 6.09002 5.30997 6.72002 5.69997 7.11002L10.59 12L5.69997 16.89C5.30997 17.28 5.30997 17.91 5.69997 18.3C6.08997 18.69 6.71997 18.69 7.10997 18.3L12 13.41L16.89 18.3C17.28 18.69 17.91 18.69 18.3 18.3C18.69 17.91 18.69 17.28 18.3 16.89L13.41 12L18.3 7.11002C18.68 6.73002 18.68 6.09002 18.3 5.71002Z"
        fill="currentColor"
        fillRule="evenodd"
      />
    </svg>
  </div>
  <div
    className="overlay"
  />
  <div
    className="snacks"
  />
  <div
    className="dialog"
  />
</div>
`;

exports[`Icons|Navigation More 1`] = `
<div
  className="container comfortable"
>
  <div
    className="container comfortable"
  >
    <svg
      height={24}
      style={
        Object {
          "color": "",
        }
      }
      viewBox="0 0 24 24"
      width={24}
    >
      <path
        clipRule="evenodd"
        d="M12 8C13.1 8 14 7.1 14 6C14 4.9 13.1 4 12 4C10.9 4 10 4.9 10 6C10 7.1 10.9 8 12 8ZM12 10C10.9 10 10 10.9 10 12C10 13.1 10.9 14 12 14C13.1 14 14 13.1 14 12C14 10.9 13.1 10 12 10ZM10 18C10 16.9 10.9 16 12 16C13.1 16 14 16.9 14 18C14 19.1 13.1 20 12 20C10.9 20 10 19.1 10 18Z"
        fill="currentColor"
        fillRule="evenodd"
      />
    </svg>
  </div>
  <div
    className="overlay"
  />
  <div
    className="snacks"
  />
  <div
    className="dialog"
  />
</div>
`;

exports[`Icons|Navigation Pagination arrows 1`] = `
<div
  className="container comfortable"
>
  <div
    className="container comfortable"
  >
    <svg
      height={24}
      style={
        Object {
          "color": "",
        }
      }
      viewBox="0 0 24 24"
      width={24}
    >
      <path
        d="M15 6.70508C14.8132 6.51783 14.5595 6.4126 14.295 6.4126C14.0305 6.4126 13.7769 6.51783 13.59 6.70508L9.00002 11.2951C8.61002 11.6851 8.61002 12.3151 9.00002 12.7051L13.59 17.2951C13.98 17.6851 14.61 17.6851 15 17.2951C15.39 16.9051 15.39 16.2751 15 15.8851L11.12 11.9951L15 8.11508C15.39 7.72508 15.38 7.08508 15 6.70508Z"
        fill="currentColor"
      />
    </svg>
    <svg
      height={24}
      style={
        Object {
          "color": "",
        }
      }
      viewBox="0 0 24 24"
      width={24}
    >
      <path
        d="M9.00002 6.71051C8.61002 7.10051 8.61002 7.73051 9.00002 8.12051L12.88 12.0005L9.00002 15.8805C8.61002 16.2705 8.61002 16.9005 9.00002 17.2905C9.39002 17.6805 10.02 17.6805 10.41 17.2905L15 12.7005C15.39 12.3105 15.39 11.6805 15 11.2905L10.41 6.70051C10.03 6.32051 9.39002 6.32051 9.00002 6.71051Z"
        fill="currentColor"
      />
    </svg>
  </div>
  <div
    className="overlay"
  />
  <div
    className="snacks"
  />
  <div
    className="dialog"
  />
</div>
`;

exports[`Icons|Navigation Subdirectory left 1`] = `
<div
  className="container comfortable"
>
  <div
    className="container comfortable"
  >
    <svg
      height={24}
      style={
        Object {
          "color": "",
        }
      }
      viewBox="0 0 24 24"
      width={24}
    >
      <path
        d="M5.00124 15.4187L9.58124 19.9987C9.97124 20.3887 10.6112 20.3887 11.0012 19.9987C11.3912 19.6087 11.3912 18.9687 11.0012 18.5787L8.12124 15.7087H18.2912C18.8412 15.7087 19.2912 15.2587 19.2912 14.7087V4.70874C19.2912 4.15874 18.8412 3.70874 18.2912 3.70874C17.7412 3.70874 17.2912 4.15874 17.2912 4.70874V13.7087H8.12124L11.0012 10.8387C11.3912 10.4487 11.3912 9.80874 11.0012 9.41874C10.6112 9.02874 9.97124 9.02874 9.58124 9.41874L5.00124 13.9987C4.61124 14.3887 4.61124 15.0287 5.00124 15.4187Z"
        fill="currentColor"
      />
    </svg>
  </div>
  <div
    className="overlay"
  />
  <div
    className="snacks"
  />
  <div
    className="dialog"
  />
</div>
`;

exports[`Icons|Navigation Subdirectory right 1`] = `
<div
  className="container comfortable"
>
  <div
    className="container comfortable"
  >
    <svg
      height={24}
      style={
        Object {
          "color": "",
        }
      }
      viewBox="0 0 24 24"
      width={24}
    >
      <path
        d="M18.9987 15.4187L14.4187 19.9987C14.0287 20.3887 13.3887 20.3887 12.9987 19.9987C12.6087 19.6087 12.6087 18.9687 12.9987 18.5787L15.8787 15.7087H5.70874C5.15874 15.7087 4.70874 15.2587 4.70874 14.7087V4.70874C4.70874 4.15874 5.15874 3.70874 5.70874 3.70874C6.25874 3.70874 6.70874 4.15874 6.70874 4.70874V13.7087H15.8787L12.9987 10.8387C12.6087 10.4487 12.6087 9.80874 12.9987 9.41874C13.3887 9.02874 14.0287 9.02874 14.4187 9.41874L18.9987 13.9987C19.3887 14.3887 19.3887 15.0287 18.9987 15.4187Z"
        fill="currentColor"
      />
    </svg>
  </div>
  <div
    className="overlay"
  />
  <div
    className="snacks"
  />
  <div
    className="dialog"
  />
</div>
`;
